grunt/helpers.js

```diff
@@ -373,6 +381,22 @@ module.exports = function(grunt) {
       });
       framework.load();
       return framework;
+    },
+
+    getLocalConfig: function() {
```

Comment:
It is for hand built courses where multiple courses need to be configured at the same time and it applies to the AAT in that one would need to create the `.adaptrc.json` file at the root of one or multiple AAT installation manually.

`scriptSafe` is string based because it was easier to configure in the AAT at the time it was implemented.